tsconfig.json

```
@@ -1,11 +1,8 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "noImplicitAny": false,
```

1. 요 옵션을 true로 설정해주세요! 

해당 옵션은 타입 추론이 되지 않을 때, `any` 타입으로 판단하는 것을 담당하는데

해당 옵션을 활성화해야 보다 타입스크립트를 사용하는 이유와 가까워지지 않을까요 ??? 👍 

```suggestion
    "noImplicitAny": true,
```

---

맞아용 😥